docs/source/common/optimizers.rst

```diff
@@ -516,3 +516,49 @@ to perform a step, Lightning won't be able to support accelerators and precision
     ):
         optimizer = optimizer.optimizer
         optimizer.step(closure=optimizer_closure)
+
+-----
+
+Configure Gradient Clipping
+---------------------------
+To configure custom gradient clipping, consider overriding
+the :meth:`~pytorch_lightning.core.lightning.LightningModule.configure_gradient_clipping` function.
+By default it will get arguments :attr:`~pytorch_lightning.trainer.Trainer.gradient_clip_val` and
+:attr:`~pytorch_lightning.trainer.Trainer.gradient_clip_algorithm` from ``Trainer`` in their respective
+fields and lightning will handle gradient clipping on its own. In case you want to set different values
```

Comment:
```suggestion
:attr:`~pytorch_lightning.trainer.Trainer.gradient_clip_algorithm` from ``Trainer`` in their respective
fields and Lightning will handle gradient clipping on its own. In case you want to set different values
```
fields?
You mean it will just get the values of the Trainer arguments as input. Can it be formulated without mentioning any "fields"?